src/page/oyblikksbilde-visning/oyeblikksbilde-egenvurdering.tsx

```diff
@@ -0,0 +1,123 @@
+import { Systemtittel } from 'nav-frontend-typografi';
+import { Alert, Link } from '@navikt/ds-react';
+import Card from '../../component/card/card';
+import OyblikksbildeType from '../../util/type/oyblikksbilde-type';
+import { useAxiosFetcher } from '../../util/use-axios-fetcher';
+import { hentEgenvurderingOyblikksbilde } from '../../api/veilarbvedtaksstotte/vedtak';
+import { useEffect } from 'react';
+import Spinner from '../../component/spinner/spinner';
+import { logMetrikk } from '../../util/logger';
+import { useViewStore, ViewType } from '../../store/view-store';
+import { formatDates } from './oyblikksbilde-fikser';
+import { EgenvurderingDto } from './dto/EgenvurderingDto';
+import { FilePdfIcon } from '@navikt/aksel-icons';
+
+export function OyeblikksbildeEgenvurdering(props: { vedtakId: number }): JSX.Element {
+	const oyeblikksbildeFetcher = useAxiosFetcher(hentEgenvurderingOyblikksbilde);
+
+	useEffect(() => {
+		oyeblikksbildeFetcher.fetch(props.vedtakId);
+		logMetrikk('vis-oyblikksbilde-egenvurdering');
+		// eslint-disable-next-line
+	}, [props.vedtakId]);
+
+	if (oyeblikksbildeFetcher.loading) {
+		return <Spinner />;
+	} else if (oyeblikksbildeFetcher.error) {
+		return (
+			<Alert variant="error" className="vedtaksstotte-alert">
+				Det oppnås for tiden ikke kontakt med alle baksystemer. Vi jobber med å løse saken. Vennligst prøv igjen
+				senere.
+			</Alert>
+		);
+	} else if (oyeblikksbildeFetcher.data) {
+		return (
+			<OyeblikksdataEgenvurderingInnhold
+				data={oyeblikksbildeFetcher.data.data}
+				erJournalfort={oyeblikksbildeFetcher.data.journalfort}
+				vedtakId={props.vedtakId}
+			/>
+		);
+	} else {
+		return <></>;
+	}
+}
+
+function OyeblikksdataEgenvurderingInnhold(props: {
+	data: EgenvurderingDto | null;
+	erJournalfort: boolean;
+	vedtakId: number;
+}) {
+	const { changeView } = useViewStore();
+
+	const visOyeblikkbildePdf = (vedtakId: number, oyeblikksbildeType: string) => {
+		changeView(ViewType.VEDTAK_OYEBLIKKSBILDE_PDF, { vedtakId: vedtakId, oyeblikksbildeType: oyeblikksbildeType });
+		logMetrikk('vis-oyeblikksbilde-vedtak', { oyeblikksbildeType: oyeblikksbildeType });
+	};
+
+	const data = props.data;
+
+	return (
+		<Card className="vedlegg-card">
+			<Systemtittel tag="h2" className="vedlegg-card__header">
+				Svarene dine om behov for veiledning
+			</Systemtittel>
+			<div className="innhold">
+				{data == null && (
+					<>
+						<b>Ingen registrerte data:</b> Personen har ikke registrert svar om behov for veiledning.
+					</>
+				)}
```

Comment:
This one doesn't seem to show up, at least on the old test users 🤔

Reply:
Yep, I saw that...i think i know what is the error